src/Explorer/Graph/NewVertexComponent/NewVertexComponent.tsx

```diff
@@ -145,13 +145,16 @@ export const NewVertexComponent: FunctionComponent<INewVertexComponentProps> = (
                     type="text"
                     id="propertyKeyNewVertexPane"
                     componentRef={input}
+                    aria-required="true"
                     placeholder="Key"
                     autoComplete="off"
                     aria-label={`Enter value for propery ${index + 1}`}
                     value={data.key}
                     onChange={(event: React.ChangeEvent<HTMLInputElement>) => onKeyChange(event, index)}
                   />
                 </div>
+                <span className="mandatoryStar">*&nbsp;</span>
```

Comment:
I don't this is necessary. It makes the UI looks a bit weird:
![image](https://user-images.githubusercontent.com/56978073/233171980-59cd4a14-4fb2-4d74-8f2a-0787b2f1e406.png)

It's unclear which field the asterisk is for. I think we should just remove it as it's not required in the bug.